### PR TITLE
feat: release note for new skins and icons

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: release-notes/release-notes52.md [hunk 1]
```diff
@@ -81,27 +81,28 @@ The {{site.productname}} 5.2 release includes **Premium Skins and Icon Packs** 1
...
 
 ### Spellchecker Pro
 
 The {{site.productname}} 5.2 release includes **Spellchecker Pro** 2.0.
 
 **Spellchecker Pro** 2.0 includes:
 
-* New functionality allowspell check the document in multiple languages
-* there's a number of new menu items/buttons
+- New functionality allowspell check the document in multiple languages
```
Comment (metricjs) [inline release-notes/release-notes52.md]: This is not a sentence?

File: release-notes/release-notes52.md [hunk 2]
```diff
@@ -81,27 +81,28 @@ The {{site.productname}} 5.2 release includes **Premium Skins and Icon Packs** 1
...
+- there's a number of new menu items/buttons
```
Comment (metricjs) [inline release-notes/release-notes52.md]: This should be worded a bit better I think? Are you waiting on details from dev?

File: release-notes/release-notes52.md (hunk 1 above)
Comment (tylerkelly13) [reply release-notes/release-notes52.md]: please ignore. the formatter in vscode changed the bullet-points. I haven't updated finished the spellchecker pro release note yet

File: release-notes/release-notes52.md (hunk 2 above)
Comment (tylerkelly13) [reply release-notes/release-notes52.md]: as previous comment, I'll doc spellchecker pro changes once the build is pushed to QA

File: release-notes/release-notes52.md (hunk 1 above)
Comment (tylerkelly13) [reply release-notes/release-notes52.md]: This will be fixed in a different PR, marking as resolved

File: release-notes/release-notes52.md (hunk 2 above)
Comment (tylerkelly13) [reply release-notes/release-notes52.md]: This will be fixed in a different PR, marking as resolved

File: release-notes/release-notes52.md (hunk 1 above)
Comment (EkimChau) [inline release-notes/release-notes52.md]: allowspell :)

Comment (tylerkelly13) [reply release-notes/release-notes52.md]: To be resolved in a different PR, please ignore

**Pull request merged**